homeassistant/components/osoenergy/strings.json

```diff
@@ -25,5 +25,50 @@
       "already_configured": "[%key:common::config_flow::abort::already_configured_device%]",
       "reauth_successful": "[%key:common::config_flow::abort::reauth_successful%]"
     }
+  },
+  "entity": {
+    "sensor": {
+      "tapping_capacity_kwh": {
+        "name": "Tapping capacity kWh"
+      },
+      "capacity_mixed_water_40": {
+        "name": "Capacity mixed water 40°C"
+      },
+      "v40_min": {
+        "name": "Mixed water at 40\u00b0C"
+      },
+      "v40_level_min": {
+        "name": "Minimum level of mixed water at 40\u00b0C"
+      },
+      "v40_level_max": {
+        "name": "Maximum level of mixed water at 40\u00b0C"
+      },
+      "heater_mode": {
+        "name": "Heater mode",
+        "state": {
+          "auto": "Auto",
+          "extraenergy": "Extra Energy",
+          "ffr": "FFR",
```

Comment:
What's a FFR?

Reply:
FFR is a special mode that responds to measurements of the frequency of the electrical grid and certain points of time. It turns off the heating elements of the water heater when the measured frequency drops below a treshold for a small amount of time to reduce the load on the grid and prevent accidents/breakdowns on the electrical grid itself. I will add the full name to the translations - Fast Frequency Reserve